### PR TITLE
MODINV-749: Add /admin/health

### PR DESCRIPTION
**Pull request by julianladisch**
https://wiki.folio.org/display/DD/Back+End+Module+Health+Check+Protocol requires /admin/health to return status code 200 (if healthy) for all back-end modules.

### CODE REVIEW

**Comment by zburke** [comment]
Sorry, @julianladisch, I don't have the technical know-how to review this PR.

File: src/main/java/org/folio/inventory/resources/AdminApi.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+public class AdminApi {
+  private static final String HEALTH_PATH = "/admin/health";
+
+  public void register(Router router) {
+    router.get(HEALTH_PATH).handler(this::health);
+  }
+
+  public void health(RoutingContext routingContext) {
+    routingContext.response().setStatusCode(200).end("OK");
```
Comment (adamdickmeiss) [inline src/main/java/org/folio/inventory/resources/AdminApi.java:14]: Content-Type = text/plain ?

**Comment by dcrossleyau** [comment]
I concur with Zak.

File: src/main/java/org/folio/inventory/resources/AdminApi.java (hunk 1 above)
Comment (julianladisch) [reply src/main/java/org/folio/inventory/resources/AdminApi.java]: Added.

**Pull request merged**